packages/boutique/backend/src/index.ts

```diff
@@ -9,10 +11,14 @@ export const start = async (app: App): Promise<void> => {
 async function bootstrap() {
   const container = await createContainer(env)
   const app = new App(container)
-  await start(app).catch(async (e): Promise<void> => {
+  try {
+    await start(app)
+    await app.processResources()
+  } catch (e) {
+    console.log('Error on starting the app')
```

Comment:
should we leave in console logs?

Reply:
yes, the app is not initialised on this step yet